internal/constant/const.go

```diff
@@ -97,6 +97,7 @@ const (
 	CMConfigurationTypeLabelKey            = "config.kubeblocks.io/config-type"
 	CMInsConfigurationHashLabelKey         = "config.kubeblocks.io/config-hash"
 	CMConfigurationConstraintsNameLabelKey = "config.kubeblocks.io/config-constraints-name"
+	CMConfigurationNewLabelKey             = "config.kubeblocks.io/new-config" // CMConfigurationNewLabelKey indicates whether this configmap is rendered by first time
```

Comment:
@sophon-zt please review these label keys, should they be better off in annotations? Do aware that label keys are usually designed to be used as a "selector" requirement.